docs/victoriametrics/changelog/CHANGELOG.md

```diff
@@ -62,6 +62,7 @@ Released at 2025-03-21
 * FEATURE: [dashboards/single](https://grafana.com/grafana/dashboards/10229), [dashboards/cluster](https://grafana.com/grafana/dashboards/11176), [dashboards/vmagent](https://grafana.com/grafana/dashboards/12683), [dashboards/vmalert](https://grafana.com/grafana/dashboards/14950): add panel `Memory allocations rate` to ResourceUsage tab, that shows the rate of allocations in memory and can help identifying issues with increased pressure on GC.
 * FEATURE: [vmbackup](https://docs.victoriametrics.com/vmbackup/), [vmrestore](https://docs.victoriametrics.com/vmrestore/), [vmbackupmanager](https://docs.victoriametrics.com/vmbackupmanager/): improve resilience to network issues by retrying requests failing due to `IncompleteBody`. Previously, such requests were not retried and leaded to restore/backup process failure. See [this PR](https://github.com/VictoriaMetrics/VictoriaMetrics/pull/8547) for details.
 * FEATURE: [vmui](https://docs.victoriametrics.com/#vmui): move legend customization settings, such as `Hide common labels` and `Table view`, closer to the legend area. This change should improve UX and make it easier for users to adjust legend visualization. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/8031)
+* FEATURE: [vmagent](https://docs.victoriametrics.com/vmagent/): now uses VM remote write protocol by default with automatic runtime downgrade to Prometheus protocol when needed. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/pull/8462) for details.
```

Comment:
Please move this line to the `tip` block above in a separate pull request, since v1.114.0 has been already released.